zeebe/bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/instance/zeebe/ZeebeBindingType.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.model.bpmn.instance.zeebe;
+
+public enum ZeebeBindingType {
+  deployment,
+  latest
```

Comment:
💭 I was surprised to see non-capitalized enum values, but they appear to be needed:
https://github.com/camunda/camunda/pull/16519#discussion_r1508047718 TIL

Reply:
I actually took `ZeebeExecutionListenerEventType` as an inspiration 🙂. But, yes, it looks a bit unusual indeed.